src/converter.js

```diff
@@ -218,10 +221,10 @@ var getHeadersArray = function(swagger, path, method) {
     for (var k in pathObj.parameters) {
       var param = pathObj.parameters[k]
       if (typeof param.in !== 'undefined' && param.in.toLowerCase() === 'header') {
-        const sample = OpenAPISampler.sample(param.schema || param, {});
+        const sample = OpenAPISampler.sample(param.schema || param, {}, swagger);
         headers.push({
           name: param.name,
-          value: JSON.stringify(sample)
+          value: encodeURIComponent((typeof sample === 'object') ? JSON.stringify(sample) : sample)
```

Comment:
Don't need to use `encodeURIComponent`. Http headers are ASCII-only

Reply:
I might actually add custom support regarding headers, at leaste for standard ones so this will be reworked

Reply:
Let's skip this for now